airbyte_cdk/sources/declarative/concurrent_declarative_source.py

```diff
@@ -206,6 +207,20 @@ def _group_streams(
             # these legacy Python streams the way we do low-code streams to determine if they are concurrent compatible,
             # so we need to treat them as synchronous
 
+            file_uploader = None
```

Comment:
One of the downside of this solution is that we need to add things on the ConcurrentDeclarativeSource level because we don't yet build a DefaultStream. This is added debt that we will clean up when we remove the declarative cursors and move the stream creation back on the model_to_component_factory but the first part is only a stretch project in Q2a so I expect that this will live there for quite some time still